doc/media/openage/modpack_definition_file.md

```diff
@@ -0,0 +1,109 @@
+`title`            | String        | Yes      | Title used in UI.
+`description`      | String        | Yes      | Path to a file with a short description (max 500 chars).
+`long_description` | String        | Yes      | Path to a file with a detailed description.
+`url`              | String        | Yes      | Link to the modpack's website.
+`license`          | Array[String] | Yes      | License(s) of the modpack.
+
+
+## [assets] Section
+
+`[assets]` contains paths to assets in the modpack.
+
+Parameter      | Data Type      | Optional | Description
+---------------|----------------|----------|------------
+`include`      | Array[String]  | No       | List of paths to assets that should be mounted on load time. Paths are allowed to contain wildcards.
+`exclude`      | Array[String]  | Yes      | List of paths to assets that should be excluded from mounting. Paths are allowed to contain wildcards.
+
+
+## [dependency] Section
+
+`[dependency]` contains a list of other modpacks that the modpack depends on.
+
+Parameter      | Data Type      | Optional | Description
+---------------|----------------|----------|------------
+`modpacks`     | Array[String]  | Yes      | List of modpack aliases or identifiers.
+
+
+## [conflict] Section
+
+`[conflict]` contains a list of other modpacks that the modpack conflicts with.
+
+Parameter      | Data Type      | Optional | Description
+---------------|----------------|----------|------------
+`modpacks`     | Array[String]  | Yes      | List of modpack aliases or identifiers.
+
+
+## [authors] Section
+
+`[authors]` contains information about the creators and authors of the modpack.
+Every author must have their own subtable `[authors.{authorname}]`. The
+subtable can set the following parameters.
+
+Parameter  | Data Type     | Optional | Description
+-----------|---------------|----------|------------
+`name`     | String        | No       | Nickname of the author.
+`fullname` | String        | Yes      | Full name of the author.
+`since`    | String        | Yes      | Version number of the release where the author started to contribute.
+`until`    | String        | Yes      | Version number of the release where the author stopped to contribute.
+`role`     | Array[String] | Yes      | List of roles of the author during the creation of the modpack.
+`contact`  | Table         | Yes      | Contact information (see below).
+
+The contact table can use the following parameters.
+
+Parameter  | Data Type     | Optional | Description
+-----------|---------------|----------|------------
+`email`    | String        | Yes      | Email address.
+`github`   | String        | Yes      | Github username.
+`gitlab`   | String        | Yes      | Gitlab username.
+`mastodon` | String        | Yes      | Mastodon username.
+`reddit`   | String        | Yes      | Reddit username.
+`twitter`  | String        | Yes      | Twitter username.
+`youtube`  | String        | Yes      | Youtube username.
```

Comment:
matrix???

Reply:
shame on me :D

Reply:
Probably the main communication platform of modders right now in the scene is Discord^^ which is missing as well.

Reply:
@simonsan True, I forgot IRC as well :( Will be fixed when the changes are integrated into the converter.